### PR TITLE
Add profile to client() to support local profiles, tweak version numb…

### PR DESCRIPTION
**Pull request by jgielstra-cs**
…er, fix env vars, added content.list() api

### CODE REVIEW

**Comment by jgielstra-cs** [comment]
- Support env vars from `cortex configure env`
- remove api version constant, just default in client
- add content list(), default content-type for uploads, 
- remove project arg from some methods ( don't require it for each call use client's )

**Review by lkrishna-cs — COMMENTED** [review]
Looks good.  Should the version (line 192 on serviceconnector) be driven by a static variable for version of the API ?

File: cortex/serviceconnector.py [hunk 1]
```diff
@@ -187,8 +187,9 @@ class _Client:
     URIs = {}
 
     def __init__(self,  *args, **kwargs):
+        # See kubernetes client for better handling of this ..  patch_namespaced_custom_object_with_http_info()  using locals() and checks..
         url = kwargs.get("url")
-        version = kwargs.get("version")
+        version = kwargs.get("version", 4)
```
Comment (lkrishna-cs) [inline cortex/serviceconnector.py:192]: Constants.default_api_version ? Can we use this instead of hardcoding

Comment (jgielstra-cs) [reply cortex/serviceconnector.py:192]: Was just defaulting, can look at re-adding a constant...  the use of @property was causing serialization issues...

**Review by laguirre-cs — APPROVED** [review]

File: cortex/serviceconnector.py [hunk 2]
```diff
@@ -187,8 +187,9 @@ class _Client:
     URIs = {}
 
     def __init__(self,  *args, **kwargs):
+        # See kubernetes client for better handling of this ..  patch_namespaced_custom_object_with_http_info()  using locals() and checks..
```
[the remark this answers is not in the document]
Comment (jgielstra-cs) [reply cortex/serviceconnector.py:190]: Just how they handle args + kwargs,  they had a pattern using locals() I wanted to investigate.https://github.com/kubernetes-client/python/blob/16935036252be0fdf164f8ce624bd71eb8d35e89/kubernetes/client/api/apps_v1_api.py#L99

- Granted this code is generated as well ..

**Pull request merged**